datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/mirrormaker/KafkaMirrorMakerConnectorTask.java

```diff
@@ -177,13 +177,13 @@ public KafkaMirrorMakerConnectorTask(KafkaBasedConnectorConfig config, Datastrea
   protected Consumer<?, ?> createKafkaConsumer(Properties consumerProps) {
     Properties properties = new Properties();
     properties.putAll(consumerProps);
-    String bootstrapValue = String.join(KafkaConnectionString.BROKER_LIST_DELIMITER,
-        _mirrorMakerSource.getBrokers().stream().map(KafkaBrokerAddress::toString).collect(Collectors.toList()));
+    String bootstrapValue = _mirrorMakerSource.getBrokers().stream().map(KafkaBrokerAddress::toString)
+        .collect(Collectors.joining(KafkaConnectionString.BROKER_LIST_DELIMITER));
     properties.putIfAbsent(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapValue);
     properties.putIfAbsent(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG,
         Boolean.FALSE.toString()); // auto-commits are unsafe
     properties.putIfAbsent(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, CONSUMER_AUTO_OFFSET_RESET_CONFIG_EARLIEST);
-    properties.put(ConsumerConfig.GROUP_ID_CONFIG,
+    properties.putIfAbsent(ConsumerConfig.GROUP_ID_CONFIG,
```

Comment:
While it makes sense to make such change, could you confirm our existing consumer settings in the server config don't reply on such override?